syfertext/doc.py

```diff
@@ -241,6 +257,7 @@ def similarity(self, other: "Doc") -> torch.Tensor:
 
         return sim
 
+    # Find Return Type: Vector
```

Comment:
same as in case of vector, methods handling vectors could be updated after we merge master into this branch.